content/client-lib-development-guide/features.textile

```diff
@@ -1464,6 +1486,110 @@ enum StatsIntervalGranularity:
   DAY
   MONTH
 
+class DeviceDetails:
+  + getDeviceInfo() => io DeviceDetails
+  id: String
+  platform: DeviceType
+  type: DeviceType
+  clientId: String?
+  metadata: Dict<String, String>
+  updateTokenId: String
```

Comment:
I prefer to think of this as a `localDeviceToken` - ie something that authenticates any request as being made by the device, and there may be multiple operations in future that are permitted to be authorised in that way; not just token renewals.